SUSE/rmt
# Updates required from testing in public cloud environment

## Description

These are the changes which have been found while testing RMT in the public cloud:

- Show a warning when migrating the database in zypper to inform users what is happening
- Move also `instance_data` to the correct place. This has been forgotten.

## How to test these changes:

- Generate a system with data in `instance_data` and migrate to the lastest database schema and check `instance_data` is still present
- Build the rpm and check that you see the data
